pkg/envoy/ads/profile.go

```diff
@@ -18,6 +19,23 @@ const (
 	MaxXdsLogsPerProxy = 20
 )
 
+// GetXDSLog implements XDSDebugger interface and a log of the XDS responses sent to Envoy proxies.
+func (s *Server) GetXDSLog() map[string]map[envoy.TypeURI][]time.Time {
```

Comment:
This map type is gnarly; may be a good candidate for a type alias?

Reply:
I agree that would be an improvement, but might pushback on that being out of scope for this PR if that's ok with everyone. Can implement if there's a strong push for it in this pr though